azure-app-service/massdriver.yaml

```
@@ -178,18 +178,12 @@ params:
       title: Container image configuration
       type: object
       required:
-      - registry
       - name
       - tag
       properties:
-        registry:
-          title: Registry
-          description: "Container registry to use for this application. For example: demo.azurecr.io"
-          type: string
-          $md.immutable: true
         name:
-          title: Image Name
-          description: "Name of the container image to use for this application. Include namespace if applicable, for example: namespace/image-name"
+          title: Image repository
+          description: "Repository of the container image to use for this application. Include namespace if applicable, for example: massdriverwestus1234.azurecr.io/namespace/image-name"
```

```suggestion
          description: "Repository of the container image to use for this application. Include namespace if applicable, for example: yourregistry.azurecr.io/namespace/image-name"
```

May want to anonymize this.